### ISSUE TITLE
Specific types are seen as wildcard types in Java

### ISSUE THREAD

**Issue by mhdhejazi** [KeenenCharles/AndroidUnplash#25]
Lists are seen in Java as lists of wildcard types, not the expected specified types. For example:
```kotlin
fun getRandomPhoto(..., onComplete: (List<Photo>) -> Unit, ...) {
}
```
The `onComplete` parameter is seen in Java as follows:
```kotlin
Function1<List<? extends Photo>, Unit> onComplete
```

I suggest adding the annotation [`@JvmSuppressWildcards`](https://kotlinlang.org/docs/reference/java-to-kotlin-interop.html#variant-generics) to fix this. It can be used on entire declarations, such as functions or classes.

**Comment by KeenenCharles** [comment]
Thanks for sharing this. I've added it and released an update (2.0.3)

**Comment by mhdhejazi** [comment]
Great! Thanks.
